src/main/java/uk/gov/justice/digital/hmpps/keyworker/dto/EliteStaffLocationRoleDto.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.justice.digital.hmpps.keyworker.dto;
```

Comment:
This is a duplicate of StaffLocationRoleDto - we don't need both.

Reply:
yes - we both added - i'll remove this